model/countries/AU/AU-model.yaml

```diff
@@ -0,0 +1,20 @@
+cut-off-children:  # don't expose nodes below these levels
+  - family-name
+  - street-address-alternative-1
+
+cut-off-tokens:
+  - honorific-prefix
+  - honorific-suffix
+  - admin-area2
+  - admin-area3
+  - admin-area4
+  - locality2
+  - locality3
+  - locality4
+
+extra-definitions:
+  street-address-alternative-1:
+    - street:
```

Comment:
`street-name` and `house-number` have no directly mapping today in Chrome. We only generate field types up to the `street` and `building` levels.  I would recommend to keep the entire `building-location` node as it would capture both concept. 

Also, reading through [1] (it was quite informative) I understood that we could also find Floors in the address, so we should probably capture that as well 

```
street-address-alternative-1:
- building-location
- in-building-location
    - floor
    - unit
```
That means we should probably add `in-building-location` to the `cut-off-children` list.

[1]https://auspost.com.au/content/dam/auspost_corp/media/documents/australia-post-addressing-standards-1999.pdf

Reply:
Adjusted the changes, as we discussed!
Thanks!